src/core/server/rendering/views/styles.tsx

```diff
@@ -53,7 +53,7 @@ export const Styles: FunctionComponent<Props> = ({ darkMode }) => {
 
           .kbnWelcomeView {
             line-height: 1.5;
-            background-color: #FFF;
+            background-color: ${darkMode ? '#1D1E24' : '#FFF'};
```

Comment:
I'm guessing there's no way to pull these from the EUI JSON theme files?

Reply:
EUI isn't loaded yet.